wicket/src/screens/component.rs

```diff
@@ -60,23 +62,44 @@ impl ComponentScreen {
         f.render_widget(block, f.size());
     }
 
-    fn draw_status_bar(&self, f: &mut Frame, state: &State) {
+    fn draw_menubar(&self, f: &mut Frame, state: &State) {
         let mut rect = f.size();
-        rect.height = 5;
+        rect.height = MENUBAR_HEIGHT;
 
-        let style = Style::default().bg(OX_GREEN_DARK).fg(OX_GRAY);
-        let selected_style = Style::default().fg(OX_GREEN_LIGHT);
-        let help_menu_style =
-            Style::default().fg(OX_OFF_WHITE).bg(OX_GREEN_DARK);
-        let help_menu_command_style =
-            Style::default().fg(OX_GREEN_LIGHT).bg(OX_GREEN_DARK);
-        let button_style = Style::default().fg(OX_OFF_WHITE).bg(OX_GREEN_DARK);
-        let hovered_style = Style::default().fg(OX_PINK).bg(OX_GREEN_DARK);
+        let bar_block = Block::default().style(style::menu_bar());
+        f.render_widget(bar_block, rect);
 
-        let status_bar_block = Block::default().style(style);
-        f.render_widget(status_bar_block, rect);
+        self.draw_component_list(f, state);
+        self.draw_power_state(f, state);
+        self.draw_help_menu(f, state);
+        self.draw_screen_selection_buttons(f, state);
+    }
+
+    fn draw_power_state(&self, f: &mut Frame, state: &State) {
+        let current = state.rack_state.get_current_component_id();
+        let title = match state.inventory.get_power_state(&current) {
+            Some(s) => {
+                format!(
+                    "⌁ Power State: {}",
+                    s.description().to_ascii_uppercase()
+                )
+            }
+            None => "⌁ Power State: UNKNOWN".to_string(),
+        };
+
+        let mut rect = f.size();
+        rect.height = 1;
+        rect.y = 3;
+        let power_state_block = Block::default()
+            .style(style::menu_bar_selected())
+            .title(title)
+            .title_alignment(Alignment::Center);
+        f.render_widget(power_state_block, rect);
+    }
 
+    fn draw_component_list(&self, f: &mut Frame, state: &State) {
         let current = state.rack_state.get_current_component_id();
+        let style = style::menu_bar();
```

Comment:
Minor nit - this variable name _seems_ like it shadows the `style` module, but I guess it doesn't given `style::` down on line 112? Maybe `menu_bar_style` would be slightly clearer?

Reply:
I agree with you 100%. I'll make this change in a follow up PR along with your suggestions from #1834.

Reply:
Actually, I have to rebase anyway, so I'll just fix this one here.